### PR TITLE
Mf12/iOsCleanUpAndCalendarImprovement

### PR DESCRIPTION
**Pull request by mwflickner**
calendars goes to week view now, refresh button for events, updated logo, links now open, logo updated, cleaning code

### CODE REVIEW

File: clients/iOS/Grapevine/Grapevine/CalendarViewController.swift [hunk 1]
```diff
@@ -40,30 +41,25 @@ class CalendarViewController: UIViewController, CVCalendarViewDelegate, CVCalend
...
-    @IBAction func weekMonthPressed(sender: UIBarButtonItem){
-        let monthStr = "Month"
-        let weekStr = "Week"
-        if sender.title == weekStr {
-            calendarView.calendarMode = .WeekView
-            sender.title = monthStr
+    @IBAction func changeView(sender: UIBarButtonItem){
+        if self.inMonthView {
+            self.toWeekView()
+            sender.title = "Month"
```
Comment (cbillingham) [inline clients/iOS/Grapevine/Grapevine/CalendarViewController.swift:47]: I might be confused but shouldn't toWeekView have sender.title = "Week". I think you may have just flipped them?

Comment (mwflickner) [inline clients/iOS/Grapevine/Grapevine/CalendarViewController.swift:47]: Nah I'm going into Week Mode but then I change the title of the button to read Month so the button can be clicked while in Week Mode to go to Month mode

Comment (cbillingham) [inline clients/iOS/Grapevine/Grapevine/CalendarViewController.swift:47]: Got it. got it. Sorry.

File: clients/iOS/Grapevine/Grapevine/HelperMethods.swift [hunk 2]
```diff
@@ -109,5 +109,12 @@ func buildEventTimeRange(myEvent: Event) -> String {
     
 }
 
+func buildEventDateString(date: CVDate) -> String {
```
Comment (cbillingham) [inline clients/iOS/Grapevine/Grapevine/HelperMethods.swift:112]: This is nice. A lot cleaner.

**Comment by cbillingham** [comment]
dude looks good to me. nice updates on this.

**Pull request merged**